src/components/overview/controlPanel/StyleSelector.tsx

```diff
@@ -25,7 +25,7 @@ const StyleSelector: React.FC<StyleSelectorProps> = (props: StyleSelectorProps)
       <div className="style-selector">
         {props.styles.results.map( (style, index) => (
           <StyleIcon style={style}
-                     key={index}
+                     key={style.style_id}
```

Comment:
I like this change, I have been trying to do something similar across my code. The index is just so easy to use.